beacon-chain/sync/backfill/batch.go

```diff
@@ -0,0 +1,141 @@
+package backfill
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/v4/consensus-types/primitives"
+	eth "github.com/prysmaticlabs/prysm/v4/proto/prysm/v1alpha1"
+	log "github.com/sirupsen/logrus"
+)
+
+// ErrChainBroken is usedbatch with no results, skipping importer when a backfill batch can't be imported to the db because it is not known to be the ancestor
```

Comment:
```suggestion
// ErrChainBroken is used for batch with no results, skipping importer when a backfill batch can't be imported to the db because it is not known to be the ancestor
```